pkg/driver/driver.go

```diff
@@ -105,6 +105,7 @@ type Info struct {
 type DriverFeatures struct {
 	CanRunGUI            bool `json:"canRunGui,omitempty"`
 	DynamicSSHAddress    bool `json:"dynamicSSHAddress"`
+	StaticSSHPort        bool `json:"staticSSHPort"`
```

Comment:
For consistency with `DynamicSSHAddress`, can we invert the logic and make this `DynamicSSHPort` ?

Reply:
Since DynamicSSHAddress doesn't really do much at the moment, we could look at revising both of them?